app/assets/stylesheets/footer.scss

```diff
@@ -4,6 +4,8 @@
 
 .footer-container-fluid {
   background-color: $footer-dark-grey;
+  position: fixed;
```

Comment:
Having `position:fixed` fixes it onto the screen on every page, try visiting any other page like groups, assignment root pages.
<img width="1470" alt="Screenshot 2025-02-03 at 6 41 29 PM" src="https://github.com/user-attachments/assets/fd9e60d6-9482-4525-97e6-3801b99908f1" />